tests/unit/AlterationSummary.spec.ts

```diff
@@ -0,0 +1,105 @@
+// Libraries
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+
+// Store
+import { getVuexStore } from '@/store'
+
+// Components
+import { createLocalVue, mount } from '@vue/test-utils'
+import { AlterationSummary } from '@/components/Summary'
+import { ConfirmDialog } from '@/components/dialogs'
+
+Vue.use(Vuetify)
+const localVue = createLocalVue()
+const vuetify = new Vuetify({})
+
+describe('AlterationSummary', () => {
+  let wrapper: any
+  let store: any = getVuexStore()
+
+  const originalSnapShot = [
```

Comment:
Why is this an array?

Reply:
The collection of the business snapshot is multiple requests, so opposed to our filings, it comes in some separate chunks before we break it down.

Reply:
Is it an array of like objects?

Reply:
I just looked it up.
What does it mean to have multiple snapshots?